accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -493,6 +493,36 @@ These are extended as per separate proposal.
 
 ## Runtime semantics
 
+### Runtime type equality operator
+
+Two objects `T1` and `T2` which are instances of `Type` (that is, runtime type
+objects) are considered equal if and only if their normal forms **NORM(`T1`)**
```

Comment:
Nitpick: This conflates `Type` objects and types. 
Maybe:
> ... if the normal forms of their corresponding types ...

Reply:
Fixed in follow up.